packages/Webkul/Shop/src/Resources/lang/en/app.php

```diff
@@ -461,6 +461,8 @@
                 'success' => 'Item was successfully added to cart.',
                 'success-remove' => 'Item was removed successfully from the cart.',
                 'error-add' => 'Item cannot be added to cart, please try again later.',
+                'inactive' => 'An item is inactive and was removed from cart.',
+                'inactive-add' => 'Inactive item cannot be added to cart.',
```

Comment:
Please add the English translation to all the other languages, to keep them up to date, even if it's in English.

Reply:
Check